content/articles/payment-methods.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: Payment methods
+excerpt: Payment methods supported by DNSimple.
+categories:
+- DNSimple
+---
+
+# Payment Methods
+
+DNSimple only supports two payment methods:
+
+## Credit card
+
+<note>
+DNSimple doesn't store credit card information. We use a PCI-compliant billing provider to process and store all payment data.
+</note>
+
+When you subscribe to DNSimple you will be required to provide a credit card. This card will be used to collect the payment for the subscription fees and all the purchases that you make in DNSimple. These purchases include:
```

Comment:
IIRC @aeden mentioned that it's not necessarily a _credit_card but could also be a debit card. If so, we should change this so only _Card_

Reply:
I'd remove the second _DNSimple_ to reduce duplication.

Reply:
✅ and ✅